ledger/sandbox/src/main/resources/db/migration/h2database/V11__Command_deduplication.sql

```diff
@@ -18,8 +18,8 @@ CREATE TABLE participant_command_submissions(
   submitted_at       timestamp             not null,
   -- The time the command will stop being deduplicated
   ttl                timestamp             not null,
-  -- If set, then the command was successfully submitted
-  success            boolean,
-  -- If set, then the command submission failed with the given error
-  error              varchar
+  -- The gRPC status code of the original command submission, if available
+  result_code        int,
```

Comment:
FYI I'm using `status_code` and `status_message` for these columns.

Reply:
Now that I think about it, can you piggy back on the work I'm doing in #4681? I'm storing completions with the intent of serving them.

Reply:
I don't think so - this the result of command submission, not command completion.

Reply:
I have renamed the columns as you suggested.